ios/sandwich/Mappers.swift

```diff
@@ -354,17 +354,48 @@ extension SKProductDiscount {
   }
 }
 
-extension Qonversion.ExperimentInfo {
+extension Qonversion.RemoteConfig {
+  func toMap() -> BridgeData {
+    return [
+      "payload": payload,
+      "experiment": experiment?.toMap()
+    ]
+  }
+}
+
+extension Qonversion.Experiment {
   func toMap() -> BridgeData {
     return [
       "id": identifier,
-      "group": [
-        "type": group?.type
-      ]
+      "name": name,
+      "group": group.toMap()
     ]
   }
 }
 
+extension Qonversion.ExperimentGroup {
+  func toMap() -> BridgeData {
+    return ["id": identifier,
+            "name": name,
+            "type": type.toString()]
+  }
+}
+
+extension Qonversion.ExperimentGroupType {
+  func toString() -> String {
+    switch self {
+    case .unknown:
+      return "unknown"
+    case .treatment:
+      return "treatment"
+    case .control:
+      return "control"
+    default:
+      return "unknown"
```

Comment:
Can we write something like this?
```suggestion
    case .treatment:
      return "treatment"
    case .control:
      return "control"
    case .unknown:
    default:
      return "unknown"
```